about.html

```diff
@@ -0,0 +1,98 @@
+        <div class="about-us">
+                          <h1>About Utop Web</h1>
+          <p>
+            We help organizations of all types and sizes prepare for the path
+            ahead — wherever it leads. Our curated collection of business and
+            technical courses help companies, governments, and nonprofits go
+            further by placing learning at the center of their strategies.
+          </p>
+        </div>
+        <div class="about-us-descrptn">
+            <div class="inner">
+
+                <h3>About Utop Web</h3>
+                <p>
+                  Utop Web was founded in 2021 with the purpose of making tech more
+                  accessible through affordable and engaging online tech education.
+                  Over the course of the following decade, we’ve taught thousands of
+                  people how to code and design. Today, Treehouse is the
+                  educational platform of choice for thousands of students, schools,
+                  libraries, non-profits, and corporations around the globe. For a
+                  glimpse into the impact Treehouse is making in the classroom, check
+                  out this video highlighting the use of our platform within a high
+                  school computer science classroom:
+                </p><br>
+                <h3>Practical, Interactive Session</h3>
+                <p>
+                  Interesting as it may be, you’re not going to find any course material covering programming self-driving cars, creating your own programming language, or building a Rubik’s Cube-solving robot. Instead, we focus exclusively on topics, languages, and technologies that are used within high-salary
+                technology roles. Programmers with knowledge of JavaScript, Angular, HTML, CSS, React, Vue and more are in extraordinarily high demand, and so that’s what we teach. We also focus on other critical parts of the web development stack, including design, user experience, databases, and product management.
+                </p>
+            </div>
+        </div>
+        </div>
+      </section>
+      <footer>
+        <div class="lets-connect">
+          <!-- <h4>Follow us on</h4> -->
+          <a href="#"><img src="./images/facebook.png" alt="">
+          </a>
+          <a href="#">
+            <img src="./images/linkedin.png" alt="LinkedIn Icon">
+        </a>
+          <a href="#">
+           <img src="./images/twitter.png" alt="LinkedIn Icon">
+          </a>
+          <a href="#">
+           <img src="./images/instagram.png" alt="LinkedIn Icon">
+          </a>
+        </div>
+        <div class="inner-sub-div">
+          <div class="footer-text">
+            <a href="#"> Terms of services</a> |
+            <a href="#"> Privacy Policy</a>
+          </div>
+          <div class="footer-text"></div>
+          <p>&copy; 2022 Utop Web</p>
+        </div>
+      </footer>
+    </main>
+  </body>
+</html>
```

Comment:
- [ ] You should also change the about page. Kindly update this page.

![image](https://user-images.githubusercontent.com/38635158/183252085-e0fd6ed0-0b50-4f0d-8fb9-4ef1da12b101.png)
![image](https://user-images.githubusercontent.com/38635158/183252090-e019831f-dc9d-4d23-ad21-20c242f9d813.png)


___
![image](https://user-images.githubusercontent.com/38635158/183252103-95ddbbb6-91e2-4d89-8138-d8c4d6fcffac.png)
![image](https://user-images.githubusercontent.com/38635158/183252113-d60c3f19-b7b8-489d-84f1-05f5634279ea.png)